crc-bundle-info.json.sample

```diff
@@ -0,0 +1,39 @@
+{
+  # Type of this bundle content
+  # Currently the only valid type is 'snc' (which stands for 'single-node-cluster')
+  "type": "snc",
+  "buildInfo": {
+    # Time this bundle was built
+    "buildTime": "2019-04-23T14:55:32+00:00",
+    # Output of 'openshift-install version' from the installer used to generate the bundle
+    "openshiftInstallerVersion": "./openshift-install v0.16.1\nbuilt from commit e3fceacc975953f56cb09931e6be015a36eb6075",
+    # Output of 'git describe' or 'git rev-parse' of the 'snc' script
+    # repository used when generating the bundle
+    "sncVersion": "git9662"
+  },
+  "clusterInfo": {
+    # Name of the openshift cluster stored in the bundle
+    "clusterName": "crc",
+    # Base domain name used for the openshift cluster
+    "baseDomain": "tt.testing",
```

Comment:
We also need the hostname which can be `${VM_PREFIX}-master-0` in our case.

Reply:
This has been added in the latest iterations.